packages/webapp/components/FooterNavBar.tsx

```diff
@@ -70,7 +70,7 @@ export default function FooterNavBar({
       className={classNames(
         'fixed !bottom-0 left-0 z-2 w-full',
         isNewMobileLayout
-          ? 'bg-gradient-to-t from-blur-baseline via-blur-bg via-70% to-transparent p-2'
+          ? 'bg-gradient-to-t from-blur-baseline via-blur-bg via-70% to-transparent p-2 pb-6'
```

Comment:
Wait is this not just for ios?
Maybe I misunderstood (also don't know how android looks, but thought only iOS has this extra space at the bottom.)

Reply:
Sab mentioned the case is the same for Android's experience. Will link the message once I am able to find it.

Reply:
https://dailydotdev.slack.com/archives/C0650M2KX8E/p1712132709640569?thread_ts=1712132480.854029&cid=C0650M2KX8E